lib/blink.js

```diff
@@ -409,24 +366,34 @@ module.exports = class Blink {
               output: process.stdout
             });
             return rl.question(`Enter the verification code sent to ${this._username}: `, pin => {
+              this._region_id = body.region.tier;
+              this._region = body.region.code;
+              this._token = body.authtoken.authtoken;
+
+              const urls = new BlinkURLHandler(body.account.id, body.region.tier);
+              headers['token-auth'] = body.authtoken.authtoken;
+
+              console.log(`url: ${urls.base_url}/api/v4/account/${body.account.id}/client/${body.client.id}/pin/verify`);
+              console.log('headers: ' + JSON.stringify(headers));
+              console.log('pin: ' + pin);
+
               request.post({
-                url: `${BASE_URL}/api/v4/account/${body.account.id}/client/${body.client.id}/pin/verify`,
+                url: `${urls.base_url}/api/v4/account/${body.account.id}/client/${body.client.id}/pin/verify`,
                 json: true,
                 headers: headers,
                 body: {
                   pin: `${pin}`,
                 }
               }, (err, response, body) => {
+                console.log('response: ' + JSON.stringify(body));
+
                 if (err || _statusCodeIsError(response)) {
                   return reject(new BlinkAuthenticationException(`Authentication problem: ${body.message}`));
                 }
-                if (!body.region) {
+                if (!valid) {
```

Comment:
I think this should be
` if (!body.valid) {
}`
because valid is not defined before.

Reply:
Should be still a check for `body.region`